packages/next/taskfile-babel.js

```diff
@@ -52,7 +52,7 @@ const babelServerOpts = {
       {
         modules: 'commonjs',
         targets: {
-          node: '8.3',
+          node: '12.0',
         },
         loose: true,
         // This is handled by the Next.js webpack config that will run next/babel over the same code.
```

Comment:
The `exclude` below this line can be removed as `async-to-generator`, `typeof-symbol`, and `spread` are supported in 12.

Reply:
Nice catch! Updated in 260ef94 👍